scripts/darwin.sh

```diff
@@ -61,6 +61,7 @@ for disk in $DISKS; do
     echo "mountpoints:"
     echo "$mountpoints" | while read -r mountpoint ; do
       echo "  - path: \"$mountpoint\""
+      echo "    volumeName: \"$mountpoint\""
```

Comment:
Huh? You're setting both `path` and `volumeName` to the same `$mountpoint` variable??